### PR TITLE
- Added function getQuantityValuesByPath

### LINKED ISSUE #1013: Update to latest core
To support GetValueByPath

### PR DESCRIPTION
**Pull request by PavelBal**
Fixes #1013

@msevestre You can review already. As soon PK-Sim gets the updated dlls, I will run styler and do a commit to trigger a build.

### CODE REVIEW

File: R/utilities-quantity.R [hunk 1]
```diff
@@ -160,7 +159,13 @@ setQuantityValuesByPath <- function(quantityPaths, values, simulation, units = N
       if (dimension == "") {
         next
       }
-      value <- toBaseUnit(quantityOrDimension = dimension, values = value, unit = units[[i]])
+      #If the unit is NULL, the value is assumend to be in base unit and no conversion
+      #in necessary
+      if (!is.null(units[[i]])){
+        mw <- simulation$molWeightFor(path)
```
Comment (PavelBal) [inline R/utilities-quantity.R:165]: Good thing the method requires a path, so we can safely use it with the performance-optimized function.

**Comment by PavelBal** [comment]
Huh, how is the build passing? :D

**Comment by msevestre** [comment]
PKSim stuff is only done on nightly builds

**Comment by msevestre** [comment]
we can merge this. The buildis working

**Pull request merged**